.github/actions/pnpm/action.yml

```diff
@@ -0,0 +1,36 @@
+
+
+name: Setup node and pnpm
```

Comment:
woah, this got bonkers -- was there a regression in pnpm support from setup-node?